tests/test_make_job_config.py

```diff
@@ -37,3 +37,13 @@ def test_container_name_can_be_set_by_caller(self):
         job_config = make_job_config(load_job_spec(), container_name='mycoolcontainer')
         container_name = job_config['spec']['template']['spec']['containers'][0]['name']
         self.assertEqual('mycoolcontainer', container_name)
+
+    def test_resources_being_passed(self):
+        test_resources = {
+                "requests": {
+                    "memory": "1Gi",
+                },
+        }
+        job_config = make_job_config(load_job_spec())
+        resource_config = job_config['spec']['template']['spec']['containers'][0]['resources']
+        self.assertEqual(resource_config, test_resources)
```

Comment:
* Naming
* By convention, expected comes first in assertions
```suggestion
        expected_resources = {
                "requests": {
                    "memory": "1Gi",
                },
        }
        job_config = make_job_config(load_job_spec())
        resource_config = job_config['spec']['template']['spec']['containers'][0]['resources']
        self.assertEqual(expected_resources, resource_config)
```